ci/local/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+# Copyright (c) 2018-2021, NVIDIA CORPORATION.
+##############################################
+# cuML local build and test script for CI    #
+##############################################
+
+
```

Comment:
Is this change needed for the PR?

Reply:
This file had originally been updated while I was debugging something unrelated. Even though I had removed the changes from the PR, CI was still telling me it needed copyright info added. I assume maybe this is because the file is still in the commit history but it didn't seem like a bad idea to add the info, since it was missing.